Makefile.kind

```diff
@@ -114,10 +121,33 @@ kind-install-cilium-clustermesh: kind-clustermesh-ready ## Install a local Ciliu
 		--set=clustermesh.apiserver.image.override=$(LOCAL_CLUSTERMESH_IMAGE) \
 		--set=clustermesh.apiserver.kvstoremesh.enabled=$(ENABLE_KVSTOREMESH)
 
-	@echo "  CONNECT the two clusters"
-	$(CILIUM_CLI) clustermesh connect --context kind-clustermesh1 --destination-context kind-clustermesh2
-	$(CILIUM_CLI) clustermesh status --context kind-clustermesh1 --wait
-	$(CILIUM_CLI) clustermesh status --context kind-clustermesh2 --wait
+	$(MAKE) kind-connect-clustermesh
+
+.PHONY: kind-install-cilium-clustermesh-fast
+kind-install-cilium-clustermesh-fast: kind-clustermesh-ready ## Install a local Cilium version into the ClusterMesh clusters and enable ClusterMesh.
```

Comment:
nit: comment is now wrong, it just loads the images not installs it

Reply:
:thinking: It uses the CLI to install Cilium right after the image load, though

Reply:
I was just trying to make the point that the comment is the same, I didn't do a great job phrasing that. If we're ok with them both being the same no problem

Reply:
oh yes you're right, updated the comment (and the one for `kind-install-cilium-fast` as well, in a separate commit)